src/Harness/Startup/SwaggerConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+namespace Harness.Startup;
+
+public static class SwaggerConfiguration
+{
+    public static WebApplication ConfigureSwagger(this WebApplication app)
+    {
+        if (!app.Environment.IsDevelopment()) return app;
```

Comment:
Always use braces for safety, your eye can be fooled when the code gets updated,
e.g.

```
if (!app.Environment.IsDevelopment()) 
        return app;
        app.UseSwagger();
        app.UseSwaggerUI();
```